docs-src/content/_index.md

```diff
@@ -2,11 +2,239 @@
-This documentation refers to accessing the network-based Cubic server.
+Cubic is Cobalt’s automatic speech recognition (ASR) engine. It can be deployed on-prem and accessed over the network or on your local machine via an API. We currently support Go and Python, and are adding support for more languages.
+
+Once running, Cubic’s API provides a method to which you can stream audio. This audio can either be from a microphone or a file. We recommend uncompressed WAV as the encoding, but support other formats such as MP3.
+
+![Cubic setup](img/cubicoverview.png)
+
+Cubic’s API provides a number of options for returning the speech recognition results. The results are passed back using Google’s protobuf library, allowing them to be handled natively by your application. Cubic can estimate its confidence in the transcription result at the word or utterance level, along with timestamps of the words. Confidence scores are in the range 0-1. Cubic’s output options are described below.
+
+## Automatic Transcription Results
+
+The simplest result that Cubic returns is its best guess at the transcription of your audio. Cubic recognizes the audio you are streaming, listens for the end of each utterance, and returns the speech recognition result. 
+
+Cubic maintains its transcriptions in an N-best list, i.e. is the top N transcriptions from the recogniser. The best ASR result is the first entry in this list.
+
+<details>
+<summary>Click here to see an example json representation of Cubic’s N-best list with utterance-level confidence scores</summary>
+
+``` json
+{
+  "alternatives": [
+    {
+      "transcript": "TOMORROW IS A NEW DAY",
+      "confidence": 0.514
+    },
+    {
+      "transcript": "TOMORROW IS NEW DAY",
+      "confidence": 0.201
+    },
+    {
+      "transcript": "TOMORROW IS A <UNK> DAY",
+      "confidence": 0.105
+    },
+    {
+      "transcript": "TOMORROW IS ISN'T NEW DAY",
+      "confidence": 0.093
+    },
+    {
+      "transcript": "TOMORROW IS A YOUR DAY",
+      "confidence": 0.087
+    }
+  ],
+}
+```
+</details>
+
+
+A single stream may consist of multiple utterances separated by silence. Cubic handles each utterance separately.
+
+For longer utterances, it is often useful to see the partial speech recognition results while the audio is being streamed. For example, this allows you to see what the ASR system is predicting in real-time while someone is speaking. Cubic supports both partial and final ASR results.
+
+## Confusion Network
+
+A Confusion Network is a form of speech recognition output that’s been turned into a compact graph representation of many possible transcriptions, as here:
+
+![Confusion Network Example](img/confnet.png)
+
+Note that `<eps>` in this representation is silence.
+
+<details>
+<summary>Click here to see an example json representation of this Confusion Network object, with time stamps and word-level confidence scores</summary>
```

Comment:
The Click here ... line kind of gets lost in the text when viewing the page. Adding a color and bold tag makes it stand out a little :
```html
<summary><font color="236ecc"><b>Click here to see an example json representation of this Confusion Network object, with time stamps and word-level confidence scores</b></font></summary>
```